hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/RepositoryApplicationConfiguration.java

```diff
@@ -264,6 +266,20 @@ public Object postProcessAfterInitialization(@NonNull final Object bean, @NonNul
         };
     }
 
+    @Bean
+    public ThreadPoolTaskExecutor rolloutTaskExecutor(
```

Comment:
It's a common utility _ThreadPoolTaskExecutor_ 'interface' but too specific - intended to be used just for rollout scheduler. 
It could happen that, registering like that, it could be injected in other beans and used.

Better create this instance into RolloutsScheduler and doesn't expose it as a bean